### DISCUSSION TITLE
Can't download song which is titled in any other language rather than English

### DISCUSSION THREAD

**Discussion by Sai-Sarat** [Shabinder/SpotiFlyer#2666] [Q&A]
Hello👋 

I've been using Spotiflyer for 3-4 months. Everything was fine.

Until, suddenly I can't download specific song titled as "ты и я"

https://open.spotify.com/track/22sQjY9bOaw2ZaIvD0WXZY?si=X7GRAPZ6QPifaNL8ysInIQ&utm_source=copy-link

I think the code is not training for this Russian characters. But only removes spaces between the words while downloading other song. So if it is please update the app to download other language titled songs.

Thanks,
Sai Sarath B

**Comment by mathi300** [comment]
Downloaded songs
